src/React/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "symfony/ux-react",
+    "type": "symfony-bundle",
+    "description": "Integration of React in Symfony",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Titouan Galopin",
+            "email": "galopintitouan@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\React\\": ""
+        },
+        "exclude-from-classmap": [
+            "/Tests/"
+        ]
+    },
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/config": "^4.4.17|^5.0|^6.0",
+        "symfony/http-kernel": "^4.4.17|^5.0|^6.0",
+        "symfony/dependency-injection": "^4.4.17|^5.0|^6.0",
+        "symfony/webpack-encore-bundle": "^1.11"
+    },
+    "require-dev": {
+        "symfony/framework-bundle": "^4.4.17|^5.0|^6.0",
+        "symfony/phpunit-bridge": "^5.2|^6.0",
+        "symfony/twig-bundle": "^4.4.17|^5.0|^6.0",
+        "symfony/var-dumper": "^4.4.17|^5.0|^6.0"
```

Comment:
Recently, for new components, we have been much more aggressive on versions. For example, Notify requires 8.1 and php 5.4: https://github.com/symfony/ux/blob/2.x/src/Notify/composer.json

Obviously, you've made this work with older versions of things. Still, I think we should require at least 8.0 and definitely 5.4 or higher.

Reply:
I solved this by relying on Webpack Encore Bundle deps